main.tf

```diff
@@ -4,12 +4,21 @@
 # Configures standard account & IAM parameters
 ##############################################################################
 
+locals {
+  # If shell_settings_enabled is false, then skip_cloud_shell_calls must be true.
+  cloud_shell_settings_validate_condition = var.shell_settings_enabled == false && var.skip_cloud_shell_calls == false
+  cloud_shell_settings_validate_msg       = "'skip_cloud_shell_calls' must be set to true when 'shell_settings_enabled = true'"
```

Comment:
I don't understand this? If we skip the cloud shell calls, the value of `shell_settings_enabled` is irrelevant

Reply:
Like cloud shell may be enabled or disabled in the account, so we can't assume its going to be enabled if we skip the check